lib/runner/extensions/ssh-command.js

```diff
@@ -0,0 +1,55 @@
+var SshTunnelAgent = require('../ssh-tunnel-agent').SshTunnelAgent,
+    _ = require('lodash'),
+    Sdk = require('postman-collection');
```

Comment:
InitialCaps is used for class names or for factory functions that return classes.